### PR TITLE
Greedy Monroe & P algorithm

### PR DESCRIPTION
**Pull request by msluszniak**

### CODE REVIEW

**Comment by msluszniak** [comment]
[Implement greedy Monroe algorithm](https://trello.com/c/oSQmCnFL/63-implement-greedy-monroe-algorithm)

File: votepy/chamberlin-courant.py [hunk 1]
```diff
@@ -30,31 +39,27 @@ def scoring_function(committee: Iterable[int], voting: OrdinalElection, number_o
...
-    return brute_force(voting, size_of_committee, lambda committee, voting: scoring_function(committee,
-                                                                                             voting, number_of_scored_candidates))
+    return brute_force(voting, size_of_committee,
+                       lambda committee, voting: k_borda(committee, voting, number_of_scored_candidates))
+
 
 
 def chamberlin_courant_greedy(voting: Union[OrdinalElection, List[int]], size_of_committee: int,
-                              number_of_scored_candidates: int) -> List[int]:
+                              number_of_scored_candidates: int, scoring_function: str = "k_borda") -> List[int]:
```
Comment (Wolokin) [inline votepy/chamberlin-courant.py]: Is scoring_function used anywhere?

File: votepy/chamberlin-courant.py [hunk 2]
```diff
@@ -63,9 +68,35 @@ def scoring_function(committee: Iterable[int], voting: OrdinalElection, candidat
...
 
+
     return greedy(voting, size_of_committee,
-                  lambda committee, voting, candidate: scoring_function(committee, voting, candidate,
-                                                                        number_of_scored_candidates))
+                  lambda committee, voting, candidate: k_borda(committee, voting, candidate,
+                                                                                   number_of_scored_candidates))
+
+
+def chamberlin_courant_p_algorithm(voting: Union[OrdinalElection, List[int]], size_of_committee: int) -> List[int]:
```
Comment (Wolokin) [inline votepy/chamberlin-courant.py:77]: It would be nice if a source for the algorithm was added

File: votepy/generic_p_algorithm.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    for idx, vote in enumerate(p):
+        if vote in C:
+            return m - idx - 1
+
+
+def simple_profile_score(P, C):
+    return sum([simple_score(p, C) for p in P])
+
+
+def p_algorithm(voting: OrdinalElection, size_of_committee: int,
```
Comment (Wolokin) [inline votepy/generic_p_algorithm.py:19]: I would add a source here as well

**Review by Wolokin — APPROVED** [review]

**Pull request closed without merging**